extensions/fairifier-rdf-extension/dtls/fairifier/GetFairDataPointInfoCommand.java

```diff
@@ -98,6 +124,20 @@ public void doPost(HttpServletRequest req, HttpServletResponse res) throws Servl
         return out;
     }
     
+    /**
+     * 
+     * Get all catalogs in a FDP based on the url of the FDP.
+     * 
+     * This method uses the FAIRMetadata4j library.
+     * 
+     * 
+     * @param url
+     * @return ArrayList<CatalogMetadata>
+     * @throws IOException
+     * @throws LayerUnavailableException
+     * @throws RDFParseException
+     * @throws RDFHandlerException
```

Comment:
It would be nice to have a description for the `@throws` definitions which describe when a certain exception can expected to be thrown.